### PR TITLE
Fix compile warning: unused variable ‘ssl_mode’ [-Wunused-variable]

### PR DESCRIPTION
**Pull request by pali**

### CODE REVIEW

**Comment by CaptTofu** [comment]
Hi there - I was looking at the latest master and it seems there is newer code that is already addressing this?

**Comment by pali** [comment]
Nope, the whole fix for SSL (CVE-2017-10789) was reverted in https://github.com/perl5-dbi/DBD-mysql/commit/6fd72e55c9771b25b5775d08a120baae3181806e.

File: dbdimp.c [hunk 1]
```diff
@@ -1935,7 +1935,10 @@ MYSQL *mysql_dr_connect(
 	    char *ca_path = NULL;
 	    char *cipher = NULL;
 	    STRLEN lna;
+  #if defined(HAVE_SSL_MODE) || defined(HAVE_SSL_MODE_ONLY_REQUIRED)
 	    unsigned int ssl_mode;
+  #endif
+	    my_bool ssl_enforce = 1;
```
[the remark this answers is not in the document]
Comment (pali) [reply dbdimp.c]: In my original May 2017 commit (https://github.com/perl5-dbi/DBD-mysql/pull/135/commits/f4f22e685db8ffdd23eaedea7b826894e9dcab9f) there is no duplication.

**Comment by dveeden** [comment]
This is fixed in master

**Pull request closed without merging**